manifests/params.pp

```diff
@@ -215,19 +215,32 @@
   $puppet_user = 'puppet'
   $puppet_group = 'puppet'
 
+  if versioncmp($::puppetversion, '4.0') < 0 {
+    $aio_package = false
+  } elsif $::osfamily == 'Windows' or $::rubysitedir =~ /\/opt\/puppetlabs\/puppet/ {
+    $aio_package = true
+  } else {
+    $aio_package = false
+  }
+
+  $puppet_ssldir = $aio_package ? {
+    true    => '/etc/puppetlabs/puppet/ssl',
```

Comment:
You're right, we don't fully support that anyway. I was thinking of settings.yaml